Doc/whatsnew/3.6.rst

```diff
@@ -162,10 +162,10 @@ Windows improvements:
 
 * A ``._pth`` file can be added to force isolated mode and fully specify
   all search paths to avoid registry and environment lookup. See
-  :ref:`the documentation <finding_modules>` for more information.
+  :ref:`the documentation <windows_finding_modules>` for more information.
 
 * A ``python36.zip`` file now works as a landmark to infer
-  :envvar:`PYTHONHOME`. See :ref:`the documentation <finding_modules>` for
+  :envvar:`PYTHONHOME`. See :ref:`the documentation <windows_finding_modules>` for
```

Comment:
Why does this PR change "Doc/whatsnew/3.6.rst"? 3.9 still gets bug fixes. 3.7 and 3.8 get security fixes, and I don't know how that relates to documentation updates. But 3.6 reached the end of its five-year release cycle several months ago (2016-12-23 to 2021-09-04).

Reply:
The label was changed, `finding_modules` was changed to `windows_finding_module`
So even though 3.6 is retired we still need the current documentation set to be internally consistent.